src/Components/SystemDetail/RecommendationsTable.js

```diff
@@ -59,7 +60,21 @@ const renderExpandedView = (row) => {
                             {suggestedInstances}</Td>
                     </Tr>
                 </Tbody>
-            </TableComposable> }
+            </TableComposable>
+            }
+            {
+                !psiEnabled &&
+                <>
```

Comment:
@r14chandra it will be good to wrap this inside  `<TextContent>` component, you can read more about it here https://www.patternfly.org/v4/components/text/ 

https://github.com/patternfly/patternfly-react/blob/83c54ab517dd329d924d69ade0430c389aaff5b1/packages/react-core/src/components/Text/TextContent.tsx

thoughts?

Reply:
This part is already under `<TextContent>`. Am I missing anything?